src/taipy/core/config/data_node_config.py

```diff
@@ -306,6 +306,12 @@ def default_config(cls):
             cls._DEFAULT_KEY, cls._DEFAULT_STORAGE_TYPE, cls._DEFAULT_SCOPE, cls._DEFAULT_VALIDITY_PERIOD
         )
 
+    def _clean(self):
+        self._storage_type = self._DEFAULT_STORAGE_TYPE
```

Comment:
@jrobinAV @trgiangdo 
For cases where we have a global config like JobConfig, clean JobConfig means cleaning all the changes to the global state. Meanwhile, here clean seems for me to be resetting a specific data node config back to the default and doesn't affect the global behavior of Taipy. Idk but the name "clean" feels kinda weird in that context. :/ For me, clean in that case should mean removing the datanode config but Idk if that makes more sense :/

Reply:
I understand your point. I had the same concern at first. We can see it in a slightly different way: 

Cleaning means removing all information from the section. Or making the section empty. 
But what does that mean to remove information on scope or make the scope empty? It means we remove the information added by the user on it. So we set the default value. Same for the storage_type, etc...

The whole purpose of this is to keep a reference to the section. so when recreating it, it is the same instance.

Reply:
hmm If you put it that way then I don't have any complaints :D it still bugs me but I'm fine with your point too :D